php/libraries/CouchDB.class.inc

```diff
@@ -199,14 +199,14 @@ class CouchDB
      *
      * @param string $database The couchDB database name
      * @param string $host     The couchDB host name
-     * @param string $port     The couchDB port number
+     * @param ?int   $port     The couchDB port number
      * @param string $user     The couchDB user name
      * @param string $password The couchDB password
      */
     private function __construct(
         string $database = '',
         string $host = '',
-        string $port = '',
+        ?int   $port = null,
```

Comment:
What does it mean to have a null port? I think this should be non-nullable and have a default of 5984 (CouchDB's default port)